magpie/api/management/user/user_utils.py

```diff
@@ -194,8 +194,10 @@ def update_user(user, request, new_user_name=None, new_password=None, new_email=
     # FIXME: disable email edit when self-registration is enabled to avoid not having any confirmation of new email
     #   (see https://github.com/Ouranosinc/Magpie/issues/436)
     update_email_admin_only = False
-    if update_email and asbool(get_constant("MAGPIE_USER_REGISTRATION_ENABLED", request)):
-        update_email_admin_only = True
+    if update_email:
+        update_email_admin_only = asbool(get_constant("MAGPIE_USER_REGISTRATION_ENABLED", request,
+                                                      default_value=False, print_missing=True,
```

Comment:
Can we prevent other similar errors by searching the full source tree for other `get_constant` missing `default_value`?

Reply:
I add them case-by-case because there are places where values are mandatory.